lib/good-influx-http.js

```diff
@@ -28,11 +67,14 @@ class GoodInfluxHttp extends GoodHttp {
             payload: lineData.join('\n')
         };
 
-        Hoek.merge(wreckOptions, this._settings.wreck, false);
+        Hoek.merge(wreckOptions, this._settings.wreck, { nullOverride: false });
 
         // Prevent this from user tampering
-        wreckOptions.headers['content-type'] = 'text/plain';
-        Wreck.request('post', this._endpoint, wreckOptions, callback);
+        wreckOptions.headers ? wreckOptions.headers['content-type'] = 'text/plain'
```

Comment:
change was needed so this error wouldn't occur:
`  26) GoodInflux HTTP URL => Sends events in a stream to HTTP server:

      Cannot set property 'content-type' of undefined

      at GoodInfluxHttp._sendMessages (/Users/solon.yang/good-influx/lib/good-influx-http.js:73:130)
      at GoodInfluxHttp._write (/Users/solon.yang/good-influx/lib/good-influx-http.js:42:102)
      at doWrite (_stream_writable.js:397:12)
      at clearBuffer (_stream_writable.js:524:7)
      at onwrite (_stream_writable.js:449:7)
      at runCallback (timers.js:810:20)
      at tryOnImmediate (timers.js:768:5)
      at processImmediate [as _immediateCallback] (timers.js:745:5)`

Reply:
Ternary assignment looks really odd to me. I'd actually rather see an if/else for an assignment.

The point of a ternary is usually get get the result.